src/main/java/gov/cms/ab2d/contracts/hmsapi/HPMSEnrollment.java

```diff
@@ -0,0 +1,47 @@
+package gov.cms.ab2d.contracts.hmsapi;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor  // Needed for Jackson
+@AllArgsConstructor
+public class HPMSEnrollment {
+    private String enrollmentYear;
+    private String enrollmentMonth;
+    private String contractId;
+    private String totalEnrollment;
+    private String medicareEligible;
+
+    public HPMSEnrollment(int enrollmentYear, int enrollmentMonth, String contractId, int totalEnrollment, int medicareEligible) {
+        this.enrollmentYear = "" + enrollmentYear;
```

Comment:
Can we just use a converter? I don't like the way of conversion. It's doing the same thing plus extra steps/